### PR TITLE
[Doc] Adding user tools info into EMR getting started guide [skip ci]

### LINKED ISSUE #7990: [DOC] Add references to qualification and bootstrap tools for EMR
I was told that we do support EMR in the qualification and bootstrap tools, yet they are not mentioned here: https://github.com/NVIDIA/spark-rapids/blob/branch-23.04/docs/get-started/getting-started-aws-emr.md. 

We should figure out if that's just a link to: https://github.com/NVIDIA/spark-rapids/blob/branch-23.04/docs/spark-qualification-tool.md and the documentation for the bootstrap tool.

### PR DESCRIPTION
**Pull request by mattahrens**
Closes #7990

### CODE REVIEW

**Comment by mattahrens** [comment]
build

**Comment by mattahrens** [comment]
build

File: docs/get-started/getting-started-aws-emr.md [hunk 1]
```diff
@@ -29,6 +29,47 @@ notes](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-6x.html).
 For more information on AWS EMR, please see the [AWS
 documentation](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html).
 
+## Leveraging Spark RAPIDS User Tools For Qualification and Bootstrap
```
[the remark this answers is not in the document]
Comment (mattahrens) [reply docs/get-started/getting-started-aws-emr.md]: Fixed in latest commit

File: docs/get-started/getting-started-aws-emr.md [hunk 2]
```diff
@@ -236,6 +277,33 @@ sudo chmod a+rwx -R /sys/fs/cgroup/cpu,cpuacct
 sudo chmod a+rwx -R /sys/fs/cgroup/devices
 ```
 
+### Running the Spark RAPIDS User Tools Bootstrap For Optimal Cluster Spark Settings
```
[the remark this answers is not in the document]
Comment (mattahrens) [reply docs/get-started/getting-started-aws-emr.md]: Fixed in latest commit

**Comment by mattahrens** [comment]
build

**Review by abellina — APPROVED** [review]

**Pull request merged**